check-requirements.in

```diff
@@ -1,7 +1,16 @@
 bandit
+darglint
```

Comment:
Verifies that the inputs (parameters) and outputs (return values, exceptions) in the docstring match the implementation.